test/mocks.h

```diff
@@ -30,20 +30,25 @@ extern "C"
 
 typedef struct mock_os_data_obj
 {
-   uint8_t     eth_send_copy[PF_FRAME_BUFFER_SIZE];
-   uint16_t    eth_send_len;
-   uint16_t    eth_send_count;
+   uint8_t                 eth_send_copy[PF_FRAME_BUFFER_SIZE];
+   uint16_t                eth_send_len;
+   uint16_t                eth_send_count;
 
-   uint16_t    udp_sendto_len;
-   uint16_t    udp_sendto_count;
+   uint16_t                udp_sendto_len;
+   uint16_t                udp_sendto_count;
 
-   uint8_t     udp_recvfrom_buffer[PF_FRAME_BUFFER_SIZE];
-   uint16_t    udp_recvfrom_length;
-   uint16_t    udp_recvfrom_count;
+   uint8_t                 udp_recvfrom_buffer[PF_FRAME_BUFFER_SIZE];
+   uint16_t                udp_recvfrom_length;
+   uint16_t                udp_recvfrom_count;
 
-   uint16_t    set_ip_suite_count;
+   uint16_t                set_ip_suite_count;
+
+   uint32_t                current_time_us;
+
+   char                    latest_filename[100];
```

Comment:
Isn't latest redundant? All existing mock data is the latest version.

Reply:
Done